src/Tenant.php

```diff
@@ -12,6 +12,7 @@ class Tenant extends Model implements TenantModel
     protected $guarded = [];
     protected $primaryKey = 'uuid';
     public $incrementing = false;
+    public $timestamps = false;
 
```

Comment:
// As recommended in Laravel docs, when the primary key is not an integer then:
`

    /**
     * The "type" of the auto-incrementing ID.
     *

     * @var string

     */

    protected $keyType = 'string';

`

Reply:
> The "type" of the auto-incrementing ID.

I don't think this is necessary, since we have set `$incrementing = false`